src/index.d.ts

```diff
@@ -602,6 +602,10 @@ declare namespace admin.auth {
      * resets, password or email updates, etc).
      */
     tokensValidAfterTime?: string;
+  
```

Comment:
These require a review from @egilmorez or somebody from his team. How about we make these changes in a separate PR, so the rest of the code changes don't get blocked on documentation updates?

Reply:
I reverted the reference changes. Will have a follow up PR just for that.